msrestazure/azure_active_directory.py

```diff
@@ -534,6 +534,7 @@ class AdalAuthentication(Authentication):  # pylint: disable=too-few-public-meth
     """
 
     def __init__(self, adal_method, *args, **kwargs):
+        super(AdalAuthentication, self).__init__()
```

Comment:
do you want to pass in the `args` and `kwargs`

Reply:
nope